workflow-controller/controller-service/src/main/java/io/hyscale/controller/validator/impl/DockerDaemonValidator.java

```diff
@@ -39,7 +41,7 @@
     private static final Logger logger = LoggerFactory.getLogger(DockerDaemonValidator.class);
 
     @Autowired
-    private HyscaleDockerClient hyscaleDockerClient;
+    private ImageCommandProvider commandProvider;
```

Comment:
These changes doesn't look right. Revert them to master code.

Reply:
Code is misplaced while merging code to my feature branch. Will revert to the master along with my required changes.